### ISSUE TITLE
Public repo fleet deployment on Proxied Rancher + downstream clusters

### ISSUE THREAD

**Issue by slickwarren** [rancher/qa-tasks#1526]

**Comment by slickwarren** [comment]
this is a low priority, may make sense to move on to a different ticket if there are higher priority items.

**Comment by rancher-max** [comment]
This ticket should focus on the public repo fleet deployment, given that the proxied rancher already exists. There will be a separate work item for creating the proxied rancher server

**Comment by slickwarren** [comment]
blocked by https://github.com/rancher/qa-tasks/issues/1668 / organizing/restructuring the rancher test repo

**Comment by slickwarren** [comment]
no longer blocked: https://github.com/rancher/qa-tasks/issues/1668#issuecomment-2641507681

**Comment by rancher-max** [comment]
The remaining work here is to create an e2e Jenkins job for this.

**Comment by lscalabrini01** [comment]
https://github.com/rancher/tests/tree/main/validation/pipeline/airgap
https://github.com/rancher/tfp-automation/blob/main/tests/infrastructure/setup_proxy_rancher_test.go

**Comment by lscalabrini01** [comment]
- [x] Create Local Proxied Rancher cluster
- [x] Create e2e Jenkins job
